packages/schematics/src/collection/ngrx/rules/add-imports-to-module.ts

```diff
@@ -16,9 +16,6 @@ import { RequestContext } from './request-context';
 
 export function addImportsToModule(context: RequestContext): Rule {
   return (host: Tree) => {
-    if (!host.exists(context.options.module)) {
```

Comment:
@FrozenPandaz - why was this removed in favor of allowing a blank `""` module path?
In fact, I recommend:

```ts
if (!context.options.module) {
  throw new Error(`Required option 'module' was not specified`);
} else if (!host.exists(options.module)) {
  throw new Error(`Specified module '${options.module}' does not exist`);
}
```

Reply:
I asserted that options are valid here before any of these rules are called: https://github.com/nrwl/nx/pull/650/files/60cc67423e235d8a9baf889985614cc60f6add28#diff-f09349e38e598f4ec3630bd791e92d80R35

I added the error for providing an empty string.